src/module/actor/character/document.ts

```diff
@@ -461,7 +461,7 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
         }
 
         // Indicate that crafting formulas stored directly on the actor are deletable
-        system.crafting = fu.mergeObject({ formulas: [], entries: {} }, system.crafting ?? {});
+        system.crafting = fu.mergeObject({ formulas: [], entries: {}, repair: { value: 5 } }, system.crafting ?? {});
```

Comment:
This allows persisted values to override the default of 5, when only rule elements should be able to modify the repair value. You might need to do `system.crafting.repair = { value: 5 };` after this line.